docs/source/basic_setup.rst

```diff
@@ -0,0 +1,113 @@
+===========
+Basic Setup
+===========
+
+SubmitCGAP is a Python based tool and requires a Python installation with version >=3.7.
+This document is intended for users who are not very familiar with the command line or Python.
+The intent is to provide detailed instructions for setting up your local environment for using SubmitCGAP.
+These instructions are intended to work with Mac OS X. Linux users are presumed advanced and Windows instructions are not available at this time but will be provided in the future.
+
+-----------------------
+Unix Command Cheatsheet
+-----------------------
+
+Using this repository and interacting with file submissions assumes some knowledge of the Unix
+filesystem and familiarity with bash. What follows is a list of the essential commands you should
+know for working with SubmitCGAP. Whenever in doubt, you can enter "man <cmd>" into the terminal to
+pull up documentation on the command. You can also do such search into your favorite search engine to
+locate the manual pages. Generally you need to know how to look around the file system and create
+directories and files.
+
+* `pwd` will "print working directory" ie: the current location in the Unix file system where the terminal is "located". Whenever you start a new terminal session, odds are you will be sent to your home directory. The `~` character is a global alias for the current active users home directory.
+* `ls` will list all files and directories in the current directory
+
+
+.. code-block:: bash
```

Comment:
Also, while I like the interface that pyenv offers, I have found it hard to get installed on my machine and I've often had to fall back to brew because things get stuck. This makes me nervous about supporting pyenv at customer sites.